spec/system/home_spec.rb

```diff
@@ -174,4 +174,32 @@ def ie_alert_box_xpath
 
     expect(page).not_to have_css(".title", text: "Featured")
   end
+
+  describe "Header Card" do
+    scenario "if there is header card with link, the link content is rendered" do
+      create(:widget_card, :header, link_text: "Link text", link_url: "consul.dev")
+
+      visit root_path
+
+      expect(page).to have_link "Link text", href: "consul.dev"
+    end
+
+    scenario "if there is header card without link, the link content is not rendered" do
+      create(:widget_card, :header, link_text: nil, link_url: nil)
+
+      visit root_path
+
+      within(".header-card") { expect(page).not_to have_link }
+    end
+
+    scenario "if there is header card without link and with text, the link content is not rendered" do
+      widget = create(:widget_card, :header, link_text_es: "Link text ES", title_es: "Title ES")
+      widget.update!(link_text: "", link_url: "")
```

Comment:
```suggestion
      create(:widget_card, :header, link_text: "", link_url: "", link_text_es: "Link ES", title_es: "ES")
```

A bit easier to understand IMHO. Previously I wasn't sure why it needed to be done in two steps nor whether the second step affected the link text defined in the first one :thinking:.